### PR TITLE
Prioritization

### PR DESCRIPTION
**Pull request by ilanman**
Would love some eyeballs on this. I feel that it's a bit verbose, so any thoughts on that front are welcome.

### CODE REVIEW

File: content/post/prioritization_meeting.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+<!--more-->
+
+Despite those hurdles, I have come to believe that the end of sprint prioritization meeting is, save for sprint planning, the most crucial project management element of a successful sprint. In this post I will discuss how I came to believe that by setting stakeholder expectations, lending credibility to the Data team's work, increasing cross functional communication with respect to what the Data team does, and strengthening the partnership between Data and the Business, sprint prioritization is among the most
+useful agile processes that our team follows.
+
+### Sprint prioritization basics
+
+The key focus areas of our scrum process are: sprint planning, daily stand ups, backlog grooming, end of sprint retro and sprint prioritization. The prioritization meeting occurs at the end of each sprint, is facilitated by either a scrum master or Data lead, and includes representatives of each function that Data supports. That is, if a function in the org has work that they would like the Data team to work on in the upcoming sprint, that function would send a representative to the meeting.
```
Comment (TryHarder01) [inline content/post/prioritization_meeting.md]: style suggestion: turn the "key areas"  into a bullet list.

Comment (TryHarder01) [inline content/post/prioritization_meeting.md]: expand "org" -> "organization"

Comment (TryHarder01) [inline content/post/prioritization_meeting.md]: For my own curiosity (and maybe other readers), how many people are in this room? is this 3-5, 10, 20?

File: content/post/prioritization_meeting.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+### Setting stakeholder expectations
+
+As mentioned, with everyone in the room representatives of each function that attend the meeting get insight into what the Data team is working on. As a result stakeholders are less likely to be confused as to why their work is not being prioritized. This reduces possible resentment or the potential for slacking analysts with extraneous requests. The prioritization meeting establishes a two-way communication between Data and the rest of the org. We acknowledge that we understand business priorities and will put a good faith effort to working on it. And if we don't include a ticket in the sprint, the onus is on us to explain why. We are all held accountable.
```
[the remark this answers is not in the document]
Comment (TryHarder01) [reply content/post/prioritization_meeting.md]: >We acknowledge that we understand business priorities and will put a good faith effort to working on it. And if we don't include a ticket in the sprint, the onus is on us to explain why. We are all held accountable.

I'd try and find a stronger verb than "acknowledge"

You might be able to drop two of those quoted sentances, and maybe add an active voice to the last line, "Transparency keeps us accountable."

File: content/post/prioritization_meeting.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+### Workload transparency
+
+If Marketing concedes that Product's A/B test analysis is the most important thing Data can work on this sprint, then when they ask why we don't have time to work on their attribution model and we point to the A/B test, the conversation is much easier since there is a built in understanding. Better yet, Marketing can look on our board (assuming project management is visible to the org) and see what we are working on. This transparency means less chance for misunderstanding around Data team priorities.
```
Comment (TryHarder01) [inline content/post/prioritization_meeting.md]: What if neither team concedes their "want" is lower priority than the other? Does Data play a role in helped them find a ranking? Any advice for how to play mediator?

File: content/post/prioritization_meeting.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+### Better quality tickets
+
+Due to the consensus-based approach and transparent nature of the prioritization meeting, each function must persuade every other function that their ticket is most important, or at least more important than another team's. This organically biases each request to be well thought out and clearly depicted because when Biz Dev needs to convince a room full of non-Biz Dev folks that their research project should be prioritized, it is in their best interest to have more than a one line ticket description. Usually the more well formulated the
```
Comment (TryHarder01) [inline content/post/prioritization_meeting.md]: I think the first sentence works better without the begining clause.
drop:
>Due to the consensus-based approach and transparent nature of the prioritization meeting
and see what that looks like.

Then it begins stronger, "Each function _must_ "

File: content/post/prioritization_meeting.md (hunk 1 above)
Comment (ilanman) [reply content/post/prioritization_meeting.md]: Agreed. I was debating about bullets vs. a long sentence. I also think bullets are more readable.

Comment (ilanman) [reply content/post/prioritization_meeting.md]: I've had success with ~ 8 people (5 is good, 3 might be too few to be meaningful), but YMMV. It's hard for me to imagine having 20 people airing their opinions in a productive way.

File: content/post/prioritization_meeting.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (ilanman) [reply content/post/prioritization_meeting.md]: Great suggestion. I didn't notice how passive it sounded.

File: content/post/prioritization_meeting.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+ticket and the better it aligns with organizational priorities, the more likely it will get prioritized. This nudges Data team tickets to be more valuable and easier to understand, minimizing (but not eliminating of course) the potential for back and forth later in the sprint. As a bonus the onus of figuring out if a request is valuable or not is no longer on the Data team, easing some pressure from the sprint planning meeting.
+
+### Appreciation for the Data Team
+
+I find that when stakeholders attend the prioritization meeting for the first time they get an eye opening glimpse into what the Data team does. Marketing may not have considered that Analysts need to think about the data warehouse infrastructure as well as statistical assumptions involved in an A/B test as well as considering the value of Brand vs. Direct marketing. And of course, Data usually has to help Finance with all kinds of work. Because Data cuts horiztonally
```
[the remark this answers is not in the document]
Comment (ilanman) [reply content/post/prioritization_meeting.md]: I hear you. Somehow it doesn't read as smoothly to me if I remove "that" in this instance, but I'll pay closer attention.

File: content/post/prioritization_meeting.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+
+### Now, prioritize!
+
+Agile [doesn't necessarily make sense]({{< ref "agile-analytics-p2.md" >}}) for every Data team, and I believe that each team should be free to amend Agile philosophy to fit their needs. Having said that I find that through setting expectations, aligning the team's work with organizational objectives, and helping to ensure high quality stakeholder requests, all while allowing flexibility for internal projects, the sprint prioritization meeting is a valuable tool in almost any Data team's project and team management repertoire.
```
Comment (TryHarder01) [inline content/post/prioritization_meeting.md]: This is cool. It's fun to see more authors cite each other's work.

**Comment by TryHarder01** [comment]
@ilanman I threw some red ink at you. That was a fun read. I found myself adding comments about things you got me curious about while reading.

File: content/post/prioritization_meeting.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+completely beholden to the business. The prioritization meeting therefore is not destiny, but instead a guide ensuring that Data's work is aligned with the organzation's. 
+
+### Does it scale?
+
+In my experience prioritization meeting works well when there are fewer than 8 active stakeholders in the room. Of coure your mileage may vary. Data is there to help facilitate the conversation and chime in with technical thoughts on implementation, potentially opining on how difficult or simple a stakeholder task is. But what happens when an organization has 100's of Data stakeholders? In my estimation nothing changes except the scope of the meeting. You can have a Marketing<>Data prioritization meeting, a Product<>Data prioritization meeting, and so on. And perhaps there is a higher level prioritization meeting with the heads of each function. This meeting would occur with less frequency, say monthly, instead of each sprint. But the process and benefits remain the same. This meeting scales.
```
[the remark this answers is not in the document]
Comment (ilanman) [reply content/post/prioritization_meeting.md]: This is based on my experience - if there is a best practice, I'd love to hear it!

**Comment by ilanman** [comment]
@sbreitenother @mikekaminsky @swiftsam If one of you has time, would appreciate a review.

File: content/post/prioritization_meeting.md [hunk 8]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Prioritizing Prioritization"
+author: "Ilan Man"
+cover: "/img/cover.jpg"
```
Comment (mikekaminsky) [inline content/post/prioritization_meeting.md]: Cover image

File: content/post/prioritization_meeting.md [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+tags: ["agile"]
+date: 2018-06-15T17:27:09-04:00
```
Comment (mikekaminsky) [inline content/post/prioritization_meeting.md]: Date

File: content/post/prioritization_meeting.md [hunk 10]
```diff
@@ -0,0 +1,64 @@
+---
+title: "Prioritizing Prioritization"
+author: "Ilan Man"
+cover: "/img/cover.jpg"
+tags: ["agile"]
```
Comment (mikekaminsky) [inline content/post/prioritization_meeting.md]: maybe also "prioritization", "project management" ?

File: content/post/prioritization_meeting.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+date: 2018-06-15T17:27:09-04:00
+draft: true
+---
+
+The sprint prioritization meeting is an integral aspect of agile philosophy. You may be more familiar with concepts like stand up, sprint planning, back log grooming and retros, but the prioritization meeting often garners less attention I suspect, partially because it is a difficult process to deploy. It involves regularly occurring meetings, thoughtful ticket (or story) descriptions, considerations of each ticket vis-a-vis every other ticket, and to top it off, this all needs to be coordinated among various Data stakeholders. There is a reason why scrum masters are typically referred to as [cat herders](https://www.google.com/search?ei=GVYpW66lCdHX5gLDsbegDg&q=%22scrum+master%22+%26+%22cat+herder%22&oq=%22scrum+master%22+%26+%22cat+herder%22&gs_l=psy-ab.3..33i160k1.5030.9792.0.10255.6.6.0.0.0.0.139.663.1j5.6.0....0...1c.1.64.psy-ab..0.1.139....0.M519ryGOesY).
```
[the remark this answers is not in the document]
Comment (mikekaminsky) [reply content/post/prioritization_meeting.md]: Here's my attempt at re-writing the whole graph

> The sprint prioritization meeting is integral to the agile process. While many people are familiar with the oft-heralded meetings like standup, sprint planning, backlog grooming, and retros, the sprint prioritization meeting often receives less attention. I suspect this is because sprint planning is a particularly challenging process to manage. A good sprint planning process requires thoughtful ticket descriptions written in advance, a collaborative review of each ticket in the context of all of the other tickets, and the buy-in and coordination among all of the analytics stakeholders.

> There is a reason why scrum masters are typically referred to as cat herders.

File: content/post/prioritization_meeting.md [hunk 12]
```diff
@@ -0,0 +1,64 @@
...
+
+<!--more-->
+
+Despite those hurdles, I have come to believe that the end of sprint prioritization meeting is, save for sprint planning, the most crucial project management element of a successful sprint. In this post I will discuss some of the benefit of this meeting, including:
```
Comment (mikekaminsky) [inline content/post/prioritization_meeting.md]: Might be useful to have  a link to a resource that lays out all of the agile meetings and what they're for.

File: content/post/prioritization_meeting.md [hunk 13]
```diff
@@ -0,0 +1,64 @@
...
+
+<li>Helping to set stakeholder expectations
+<li>Increasing transparency into Data team priorities
+<li>Improving ticket/story quality
+<li>Visibility into what analysts do
+
+Hopefully by the end you will consider implementing a sprint prioritization meeting for your team.
```
Comment (mikekaminsky) [inline content/post/prioritization_meeting.md]: Somewhere in this top section, it might be useful to have some motivating context. Why do we need this prioritization meeting in addition to all of the other meetings? What goes wrong if you don't have it?

File: content/post/prioritization_meeting.md [hunk 14]
```diff
@@ -0,0 +1,64 @@
...
+
+### Sprint prioritization basics
+
+The key focus areas of our scrum process are: sprint planning, daily stand ups, backlog grooming, end of sprint retro and sprint prioritization. The hour-long prioritization meeting occurs at the end of each bi-weekly sprint, is facilitated by either a scrum master or Data lead, and includes representatives of each function that Data supports. That is, if a function in the organization has work that they would like the Data team to work on in the upcoming sprint, that function would send a representative to the
```
Comment (mikekaminsky) [inline content/post/prioritization_meeting.md]: I'd consider dropping this paragraph, except for the last sentence. (Or re-working to emphasize who is coming to the meeting and what the purpose is). I feel like the lede is buried here.

File: content/post/prioritization_meeting.md [hunk 15]
```diff
@@ -0,0 +1,64 @@
...
+meeting. Others are welcome to sit in or dial in, but at minimum some stakeholder representation is required.
+
+The purpose of the meeting is essentially for stakeholders to work with Data to come up with a rank-ordered list of tickets for consideration in Data's next sprint. The entire group must come to a consensus around which ticket should be ranked first, second, etc... The meeting facilitator literally moves tickets up and down the board and the group decides (debates?) where a ticket should land. This way the Data team's workload is aligned with the organizations priorities without Data having to chase down stakeholders and decide what is the most important initiative this sprint. If Marketing and Product disagree on what is most important, the entire room together can weigh in with the end result being one ticket is ranked above the other. Data should opine on the technical feasibility or difficulty of executing certain tasks but in general should be unbiased participants. Data should act as tie breakers, not king makers. When prioritization works well, it is a thing of beauty for some of the following reasons.
```
Comment (mikekaminsky) [inline content/post/prioritization_meeting.md]: > When prioritization works well, it is a thing of beauty for some of the following reasons.

Either change the period to colon, or rewrite to something like

> I discuss the benefits of a good prioritization process below.

(Or consider striking the sentence altogether.)

File: content/post/prioritization_meeting.md [hunk 16]
```diff
@@ -0,0 +1,64 @@
...
+
+### Setting stakeholder expectations
+
+Representatives of each function that attend the meeting get insight into what the Data team is working on. As a result stakeholders are less likely to be confused as to why their work is not being prioritized. This reduces possible resentment or the potential for slacking analysts with extraneous requests. The prioritization meeting establishes a two-way communication between Data and the rest of the organization. The final queue is an acknowledgement that if Data doesn't include a ticket in the sprint, the onus is on us to explain why. Transparency keeps us accountable.
```
Comment (mikekaminsky) [inline content/post/prioritization_meeting.md]: "slacking" is not quite a universally accepted replacement for "contacting" (yet) :)

Comment (mikekaminsky) [reply content/post/prioritization_meeting.md]: >  Transparency keeps us accountable.

^^ good sentence! This graph could be improved if the first sentence were as direct and clean as the last one.

File: content/post/prioritization_meeting.md [hunk 17]
```diff
@@ -0,0 +1,64 @@
...
+
+### Workload transparency
+
+If Marketing concedes that Product's A/B test analysis is the most important thing Data can work on this sprint, then when they ask why we don't have time to work on their attribution model and we point to the A/B test, the conversation is much easier since there is a built in understanding. Better yet, Marketing can look on our board (assuming project management is visible to the organization) and see what we are working on. This transparency means less chance for misunderstanding around Data team priorities.
```
Comment (mikekaminsky) [inline content/post/prioritization_meeting.md]: built-in

Comment (mikekaminsky) [reply content/post/prioritization_meeting.md]: I'm sort of getting lost in this paragraph -- I think without context of Marketing and Product's priorities, it's hard to understand what's going on here. Maybe you can re-write this at a slightly higher level of abstraction? (I think that will help generalize to people who don't work at tech-company start-ups as well.)

File: content/post/prioritization_meeting.md [hunk 18]
```diff
@@ -0,0 +1,64 @@
...
+
+### Better quality tickets
+
+Each function must persuade every other function that their ticket is most important or at least more important than another team's. This organically biases each request to be well thought out and clearly depicted because when CustOps needs to convince a room full of non-CustOps folks that their Zendesk integration should be prioritized, it is in their best interest to have more than a one line ticket description. Usually the more well formulated the ticket and the better it aligns with organizational priorities, the more likely it will get prioritized. This nudges Data team tickets to be more valuable and easier to understand, minimizing (but not eliminating of course) the potential for back and forth later in the sprint. As a bonus the onus of figuring out if a request is valuable or not is no longer on the Data team, easing some pressure from the sprint planning meeting.
```
Comment (mikekaminsky) [inline content/post/prioritization_meeting.md]: As a bonus, the onus

File: content/post/prioritization_meeting.md [hunk 19]
```diff
@@ -0,0 +1,64 @@
...
+
+### Appreciation for the Data Team
+
+When stakeholders attend the prioritization meeting for the first time they get an eye opening glimpse into what the Data team does. Marketing may not have considered Analysts need to think about the data warehouse infrastructure as well as statistical assumptions involved in an A/B test as well as considering the value of Brand vs. Direct marketing. And of course, Data usually has to help Finance with all kinds of work. Because Data cuts horizontally across an organization, the breadth of work that falls in Data's scope can be surprising for stakeholders who are used to focusing on a slice of the organization.
```
Comment (mikekaminsky) [inline content/post/prioritization_meeting.md]: for the first time, they

Comment (mikekaminsky) [reply content/post/prioritization_meeting.md]: Or rephrase to
> Stakeholders attending the prioritization meeting for the first time get an eye-opening glimpse into how much the Data team does.

File: content/post/prioritization_meeting.md [hunk 20]
```diff
@@ -0,0 +1,64 @@
...
+
+### It gets more efficient
+
+While the idea of adding yet another meeting to everyone's already busy schedule is the last thing anyone wants, in reality after a few iterations, with a competent meeting facilitator, it often runs very quickly. For example it is common for our prioritization meetings to run in under 30 minutes. The more exposure teams have with each other, the easier it is to understand how to efficiently prioritize requests. Stakeholders are incentivized to pick their battles. And in practice since the top few requests are likely to be worked on, those tickets that fall beneath the "will probably work on" threshold are for the most part
```
Comment (mikekaminsky) [inline content/post/prioritization_meeting.md]: First sentence is a bit cloudy. Maybe rephrase / split up.

Comment (mikekaminsky) [reply content/post/prioritization_meeting.md]: > to run in under 30 minutes 

Maybe strike the `in`? This phrasing is a bit odd to me.

File: content/post/prioritization_meeting.md (hunk 8 above)
Comment (ilanman) [reply content/post/prioritization_meeting.md]: Will update when Margarita sends me the illustration. She's working on it.

File: content/post/prioritization_meeting.md (hunk 11 above)
[the remark this answers is not in the document]
Comment (ilanman) [reply content/post/prioritization_meeting.md]: You don't like `vis-a-vis`? :)  The re-write sounds good.

File: content/post/prioritization_meeting.md (hunk 13 above)
Comment (ilanman) [reply content/post/prioritization_meeting.md]: Agreed. I added a couple of sentences to motivate.

File: content/post/prioritization_meeting.md (hunk 14 above)
Comment (ilanman) [reply content/post/prioritization_meeting.md]: I removed the first sentence which seemed unnecessary. I think that helps.

File: content/post/prioritization_meeting.md (hunk 15 above)
Comment (ilanman) [reply content/post/prioritization_meeting.md]: Struck

File: content/post/prioritization_meeting.md (hunk 16 above)
Comment (ilanman) [reply content/post/prioritization_meeting.md]: Gave it a shot

File: content/post/prioritization_meeting.md (hunk 17 above)
Comment (ilanman) [reply content/post/prioritization_meeting.md]: I tried to be more explicit.

**Comment by mikekaminsky** [comment]
Really like this! Nice work, Ilan :)
